05_create_install_config.sh

```diff
@@ -14,7 +14,7 @@ early_deploy_validation
 
 # NOTE: This is equivalent to the external API DNS record pointing the API to the API VIP
 if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
-    if [[ -z "${EXTERNAL_SUBNET_V4}" ]]; then
+    if [[ -z "${EXTERNAL_SUBNET_V4}" || ${IP_STACK} = "v6" ]]; then
```

Comment:
That is a confusing combination of checks...

It seems like just `[[ "${IP_STACK}" == "v6" ]]` should suffice anyway. That is basically what is intended here; only create an IPv6 VIP if we're creating a single-stack IPv6 cluster (since dev-scripts doesn't support "IPv6-primary dual-stack".)